apps/admin-gui/src/app/users/pages/user-detail-page/user-settings/user-roles/user-roles.component.html

```diff
@@ -1,8 +1,341 @@
-<mat-accordion>
+<h1 class="page-title d-flex">{{'ROLES.TITLE'| translate}}</h1>
+<mat-spinner *ngIf="outerLoading" class="ml-auto mr-auto"></mat-spinner>
+<mat-accordion *ngIf="!outerLoading">
+  <mat-expansion-panel (opened)="getSelfData()">
+    <mat-expansion-panel-header>
+      <mat-panel-title>
+        <mat-icon mat-card-avatar svgIcon='perun-user-dark' class="mr-2"></mat-icon>
+        <p class="mt-auto mb-auto">{{'ROLES.SELF' | translate}}</p>
+      </mat-panel-title>
+    </mat-expansion-panel-header>
+    <mat-spinner *ngIf="loading" class="ml-auto mr-auto"></mat-spinner>
+    <div *ngIf="!loading">
+      {{'ROLES.SELF_VOS' | translate}}
+      <perun-web-apps-vo-select-table
+        [displayedColumns]="['id', 'name', 'shortName']"
+        [vos]="vos">
+      </perun-web-apps-vo-select-table>
+
+      <div class="mt-5" *ngIf="users.length">
+        {{'ROLES.SELF_IDENTITIES' | translate}}
+        <app-users-list
+          [users]="users"
+          [disableRouting]="true"
+          [displayedColumns]="['user', 'id', 'name']">
+        </app-users-list>
+      </div>
+    </div>
+  </mat-expansion-panel>
+
+  <mat-expansion-panel (opened)="getAdminGroupsAndVos()" *ngIf="isGroupAdmin">
+    <mat-expansion-panel-header>
+      <mat-panel-title>
+        <mat-icon mat-card-avatar svgIcon='perun-group-black' class="mr-2"></mat-icon>
+        <p class="mt-auto mb-auto">{{'ROLES.GROUPADMIN' | translate}}</p>
+      </mat-panel-title>
+    </mat-expansion-panel-header>
+    <mat-spinner *ngIf="loading" class="ml-auto mr-auto"></mat-spinner>
+    <div *ngIf="!loading">
+      {{'ROLES.GROUPADMIN_GROUPS' | translate}}
+      <perun-web-apps-groups-list
+        [groups]="groups"
+        [hideColumns]="['select', 'menu', 'expiration']"
+      ></perun-web-apps-groups-list>
+
+      <div class="mt-5">
+        {{'ROLES.GROUPADMIN_VOS' | translate}}
+        <perun-web-apps-vo-select-table
+          [displayedColumns]="['id', 'name', 'shortName']"
+          [vos]="vos">
+        </perun-web-apps-vo-select-table>
+      </div>
+    </div>
+  </mat-expansion-panel>
```

Comment:
![image](https://user-images.githubusercontent.com/32871167/104292246-e0617400-54bc-11eb-9717-8a11b741e9ec.png)
The table of the organizations is really unnecessary because we show the names of the VOs in the table with groups, please remove it.